windows_containers/windows-containers-release-notes-6-x.adoc

```diff
@@ -28,6 +28,25 @@ For more information, see the Red Hat OpenShift Container Platform Life Cycle Po
 If you do not have this additional Red Hat subscription, you can use the Community Windows Machine Config Operator, a distribution that lacks official support.
 endif::openshift-origin[]
 
+[id="wmco-6-0-1"]
+== Release notes for Red Hat Windows Machine Config Operator 6.0.1
+
+This release of the Windows Machine Config Operator (WMCO) provides bug fixes for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 6.0.0 were released in link:https: //access.redhat.com/errata/RHSA-2023:4488[RHSA-2023:4488].
```

Comment:
I didn't see an updated preview, so had to build it on my machine. This link doesn't work, I think because there is an extra space before the `//`. See screenshot.
![Screenshot 2023-08-15 at 3 09 55 PM](https://github.com/openshift/openshift-docs/assets/96192119/6e84353f-a391-4f22-8481-948d6a3e1726)